src/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.ts

```diff
@@ -110,7 +110,7 @@ export class CaseSpecificAccessRequestComponent implements OnDestroy, OnInit {
 
   public onCancel(): void {
     // Navigate to the page before previous one (should be Search Results or Case List page, for example)
-    window.history.go(-2);
+    this.router.navigateByUrl(`/work/my-work/list`);
```

Comment:
Ideally this should be coming from the Config URL.  That could be a tech debt for later.
Please create a constant for now rather than string.